djaopsp/management/commands/generate_test_data.py

```diff
@@ -99,7 +99,8 @@ def generate_frozen_samples(self, profiles, campaign, verification_campaign,
             enumeratedquestions__campaign=campaign,
             enumeratedquestions__question__path__startswith=segment_path)
         for profile in profiles:
-            created_at = datetime_or_now(fake.date_between(start_at, ends_at))
+            created_date = fake.date_between(start_at, ends_at)
```

Comment:
I assume this is a workaround because the [fix to survey.queries.datetime_or_now](https://github.com/djaodjin/djaodjin-survey/commit/92598a9a791c3e7d2648a1fc7e682a218ed73558) has not been part of an official release yet.